tokenserver-db-postgres/src/orm_models.rs

```diff
@@ -0,0 +1,35 @@
+use crate::schema::{nodes, services, users};
+use diesel::{Identifiable, Insertable, Queryable};
+
+#[derive(Queryable, Debug, Identifiable, Insertable)]
+pub struct Service {
```

Comment:
How do these differ from the types defined in `tokenserver-db-common::results`? It seems like those could be utilized instead -- unless we begin straying the postgres schema from the mysql one (which we haven't, yet anyway, I do wonder if we should revisit that, like e.g. using postgres `TIMESTAMP` types, but we can change our minds a little later while its still a WIP).

Reply:
Diesel uses these models in relation to the `table!` march that's defined in `schema.rs` to construct table aliases and make ORM operations possible. You have to match them up as the compiler gets upset if the plural schema def does not match the singular model name. Then, within a function to make the actual call, you pull in Ex. `use self::schema::users::dsl::*;` and that allows for various ORM calls within said method. This can always move into another crate, doesn't have to be its own. It can be a part of results, or rather even be renamed models.rs when we change the current models.rs (which we agreed isn't the greatest name currently)